Signal/src/ViewControllers/ConversationView/Cells/OWSUnreadIndicatorCell.m

```diff
@@ -135,14 +126,9 @@ - (NSString *)subtitleForInteraction:(TSUnreadIndicatorInteraction *)interaction
                   @"Messages that indicates that there are more unseen messages including safety number changes."));
 }
 
-- (CGFloat)strokeHeight
-{
-    return 1.f;
-}
-
-- (CGFloat)subtitleVSpacing
+- (CGFloat)strokeThickness
 {
-    return 3.f;
+    return 1.f / UIScreen.mainScreen.scale;
```

Comment:
TIOLI: Should we extract a free function like `CGHairlineWidth()`?

Reply:
Nice, done.